Vagrantfile

```diff
@@ -0,0 +1,39 @@
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+    sudo apt-get update
+
+    sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+    xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+
+    git clone https://github.com/pyenv/pyenv.git /home/vagrant/.pyenv 
+    echo 'export PYENV_ROOT="/home/vagrant/.pyenv"' >> /home/vagrant/.profile
+    echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> /home/vagrant/.profile
+    echo 'eval "$(pyenv init -)"' >> /home/vagrant/.profile
+    export PYENV_ROOT="/home/vagrant/.pyenv"
+    export PATH="$PYENV_ROOT/bin:$PATH"
+    eval "$(pyenv init -)"
```

Comment:
This is fine, but rather than repeating yourself, you could run the updated .profile with `source ~/.profile` or `/home/vagrant/.profile`